File: gamsojji/backjoon/step3/2739.js
let fs = require("fs");
const filePath = process.platform === 'linux' ? "/dev/stdin" : `${__dirname}/input.txt`;
let input = fs.readFileSync(filePath).toString().trim().split(' ').map(Number);

for (let i = 1; i <= 9; i++) {
    console.log(input[0]+" * "+i+" = "+input[0]*i);
}

File: rolemadelen/baekjoon/step3/2739.rs
use std::io::stdin;

fn main() {
    let mut input = String::new();
    stdin().read_line(&mut input).ok();
    let n: i32 = input.trim().parse().unwrap();

    for i in 1..10 {
        println!("{n} * {i} = {}", n*i);
    }
}
